simphony_ui/ui.py

```diff
@@ -164,6 +165,12 @@ class Application(HasStrictTraits):
                             "current_frame_index < len(frames) "
                             "and play_timer is None"),
                     ),
+                    UItem(
+                        name="last_button",
+                        enabled_when=(
+                            "current_frame_index < len(frames) "
```

Comment:
You want to make the last button disabled when you reach the end of the frame list, so that you can't go further (nothing will happen, but it makes little sense anyway)